### DISCUSSION TITLE
installing UC on windows

### DISCUSSION THREAD

**Discussion by chiragwad** [Eloston/ungoogled-chromium#1495] [Q&A] [answered]
what is the safest way to install ungoogled chromium on windows if i do not how to compile it from source?
choclatey
installing binary(which one?)
or any other way?

**Comment by tangalbert919** [comment, marked as answer]
There are binaries available from [this website](https://ungoogled-software.github.io/ungoogled-chromium-binaries/), but it can also be installed from [Chocolatey](https://community.chocolatey.org/packages/ungoogled-chromium).

**Reply by valentinpi** [reply]
Just wanted to add that you can also use [scoop](https://scoop.sh/). However, it currently won't run directly unless you run it from the version directory, `C:\Users\...\scoop\apps\ungoogled-chromium\89.0.4389.114-1` for instance.
